pytorch_lightning/core/lightning.py

```diff
@@ -1964,8 +1964,14 @@ def model_size(self) -> float:
         """
         The model's size in megabytes. The computation includes everything in the
         :meth:`~torch.nn.Module.state_dict`, i.e., by default the parameteters and buffers.
+
+        .. deprecated::v1.4
+            Will be removed in v1.6.0
         """
-        # todo: think about better way without need to dump model to drive
+        rank_zero_deprecation(
+            'The `LightningModule.model_size` property was deprecated in v1.4 and will be removed in v1.6.'
+        )
+
         tmp_name = f"{uuid.uuid4().hex}.pt"
```

Comment:
Should we move this a utility function instead ?

Reply:
Should I move it to a utility function in this PR or create a new PR?

Reply:
I think it would be reasonable to do it in this PR because we would have to update the deprecation message anyway (saying that it has been moved to ...)

Reply:
there is a parallel PR for this in https://github.com/PyTorchLightning/pytorch-lightning/pull/8500 - should we merge that and then update the message here or deprecation?